test/dummy/app/views/comboboxes/custom_attrs.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= combobox_tag "state", State.all do |combobox| %>
+  <% combobox.customize :fieldset, class: "custom-class--fieldset", data: { customized_fieldset: "" } %>
```

Comment:
Thinking I like `customize_fieldset` better. Should be simple to metaprogram. Drawback is ambiguous method source.